packages/desktop-electron/package.json

```diff
@@ -7,10 +7,12 @@
   "scripts": {
     "clean": "rm -rf dist",
     "update-client": "bin/update-client",
-    "build": "electron-builder",
-    "watch": "cross-env ACTUAL_DOCUMENT_DIR=\"../../data\" ACTUAL_DATA_DIR=\"../../data\" electron ."
+    "build": "yarn build:dist && electron-builder",
+    "build:dist": "rm -rf electron-dist && tsc --p tsconfig.dist.json && yarn copy-assets",
+    "copy-assets": "copyfiles --exclude 'client-build/**/*' **/*.html electron-dist",
```

Comment:
were there bash scripts that were doing file copies? Maybe just use existing scripts